### ISSUE TITLE
[Bug] Accessibility interaction with the slider

### ISSUE THREAD

**Issue by ismith78** [xamarin/Xamarin.Forms#14888]
### Description
I have a page that uses the Xamarin slider. It all works until I start using accessibility (TalkBack on Android, VoiceOver on iOS or VoiceView on Kindle)

Although the slider is seen in the accessibility view, if I try to drag the value to change the value either nothign happens, or the slider is stuck at 50% and cannot be moved.

To rule out our code I have tried the slider sample code  at xamarin-forms-samples-main/UserInterface/SliderDemos/SliderDemos.sln and that shows the same behaviour.

One option I have considered is to add plus and minus buttons on the page that can be used to change the value when in accessibility. However, is there another option that I am missing.

### Steps to Reproduce

1. Download the xamarin-samples from GitHub
2. Open the solution at xamarin-forms-samples-main/UserInterface/SliderDemos/SliderDemos.sln
3. Run the code on a proper Android, iOS or Kindle device
4. Enable accessibility for the platform
5. Switch back to the running app
6. Select Basic Slider Code from the options
7. Try to interact with the slider to change text

### Expected Behavior
It should be possible to interact with the slider when accessibility voiceover is enabled.

### Actual Behavior
It is not possible to interact with the slider when accessibility is enabled. The best that can happen is that the slider is stuck at 50%

### Basic Information

- Version with issue: 4.8.0, 5.0.0.1874, 5.0.0.2244
- Last known good version: Haven't found one yet and can't downgrade much below 4.8.0 due to other dependencies in our code.
- Platform Target Frameworks: 
  - iOS:  15.0
  - Android: 11 
  - UWP: N/A
- Android Support Library / AndroidX Version: 
- NuGet Packages:
- Affected Devices:
   - Honor View 10 running Android 10
   - Samsung A70 running Android 11
   - Kindle Fire 7 HD running FireOS 5.6.9.0
   - PHONEMAX X1 running Android 10
   - Samsung S6 running Android 7
   - Samsung A20 running Android 10
   - multiple iOS simulators running iOS 15.0

### Environment

<details><summary>Show/Hide Visual Studio info</summary>

```
=== Visual Studio Community 2019 for Mac ===

Version 8.10.10 (build 8)
Installation UUID: 89929244-d9d1-44d6-8132-130bf5ad66ac
	GTK+ 2.24.23 (Raleigh theme)
	Xamarin.Mac 6.18.0.23 (d16-6 / 088c73638)

	Package version: 612000140

=== Mono Framework MDK ===

Runtime:
	Mono 6.12.0.140 (2020-02/51d876a041e) (64-bit)
	Package version: 612000140

=== Roslyn (Language Service) ===

3.10.0-4.21269.26+029847714208ebe49668667c60ea5b0a294e0fcb

=== NuGet ===

Version: 5.9.0.7134

=== .NET Core SDK ===

SDK: /usr/local/share/dotnet/sdk/5.0.401/Sdks
SDK Versions:
	5.0.401
	3.1.413
MSBuild SDKs: /Applications/Visual Studio.app/Contents/Resources/lib/monodevelop/bin/MSBuild/Current/bin/Sdks

=== .NET Core Runtime ===

Runtime: /usr/local/share/dotnet/dotnet
Runtime Versions:
	5.0.10
	3.1.19

=== .NET Core 3.1 SDK ===

SDK: 3.1.413

=== Xamarin.Profiler ===

Version: 1.6.15.68
Location: /Applications/Xamarin Profiler.app/Contents/MacOS/Xamarin Profiler

=== Updater ===

Version: 11

=== Apple Developer Tools ===

Xcode 13.0 (19234)
Build 13A233

=== Xamarin.Mac ===

Version: 7.14.0.27 (Visual Studio Community)
Hash: 2566861a9
Branch: d16-10
Build date: 2021-07-27 02:34:12-0400

=== Xamarin.iOS ===

Version: 15.0.0.6 (Visual Studio Community)
Hash: 2771277e0
Branch: xcode13-ios
Build date: 2021-09-23 10:36:08-0400

=== Xamarin Designer ===

Version: 16.10.0.119
Hash: 36a2d986f
Branch: remotes/origin/d16-10
Build date: 2021-06-02 19:41:34 UTC

=== Xamarin.Android ===

Version: 11.3.0.4 (Visual Studio Community)
Commit: xamarin-android/d16-10/ae14caf
Android SDK: /Users/ian/Library/Developer/Xamarin/android-sdk-macosx
	Supported Android versions:
		None installed

SDK Tools Version: 26.1.1
SDK Platform Tools Version: 30.0.4
SDK Build Tools Version: 30.0.2

Build Information: 
Mono: b4a3858
Java.Interop: xamarin/java.interop/d16-10@f39db25
ProGuard: Guardsquare/proguard/v7.0.1@912d149
SQLite: xamarin/sqlite/3.35.4@85460d3
Xamarin.Android Tools: xamarin/xamarin-android-tools/d16-10@c5732a0

=== Microsoft OpenJDK for Mobile ===

Java SDK: /Users/ian/Library/Developer/Xamarin/jdk/microsoft_dist_openjdk_1.8.0.25
1.8.0-25
Android Designer EPL code available here:
https://github.com/xamarin/AndroidDesigner.EPL

=== Android SDK Manager ===

Version: 16.10.0.13
Hash: 1b81df5
Branch: remotes/origin/d16-10
Build date: 2021-09-21 02:30:50 UTC

=== Android Device Manager ===

Version: 16.10.0.15
Hash: 89dcc0b
Branch: remotes/origin/d16-10
Build date: 2021-09-21 02:31:08 UTC

=== Build Information ===

Release ID: 810100008
Git revision: a3ff4b6e658e1f94623e1f3ed34ca94ed4fe78d8
Build date: 2021-09-23 19:50:51-04
Build branch: release-8.10

=== Operating System ===

Mac OS X 10.16.0
Darwin 20.6.0 Darwin Kernel Version 20.6.0
    Tue Oct 12 18:33:42 PDT 2021
    root:xnu-7195.141.8~1/RELEASE_X86_64 x86_64

```

</details>

### Screenshots

![image](https://user-images.githubusercontent.com/19591171/141978972-0acc7e65-45bc-40f4-836e-29bc253562c6.png)

### Reproduction Link

1. Start the basic slider code project from xamarin-forms-sample-main
2. Open the UserInterface/SliderDemo project
3. Deploy the code to a working real Android, iOS or Kindle device
4. Once running enable the device's accessibility tool
5. Switch back to the running app
6. Attempt to change the rotation or the RBG value of the slider with accessibility enabled

**Comment by ismith78** [comment]
Any update?

**Comment by rachelkang** [comment]
Hi, @ismith78 - I tested on an Android device and was able to interact with the slider in TalkBack mode. 
There are two ways I discovered, both of which are consistent to the Android platform experience as well.

While the slider is focused in TalkBack mode, you can try a two finger horizontal slide or one finger vertical flick. Let me know if this works for you!

**Comment by ismith78** [comment]
Thank you @rachelkang for your feedback. I've just tested that on my A70 and it does for me with that two finger slide.

Does this work the same way on iOS? I'm guessing you don't have a Kindle to test with, so I'll check that tomorrow.

**Comment by rachelkang** [comment]
I explored iOS a bit more, and it seems there is indeed something similar.

In addition to the one finger vertical swipe up/down, you can try to double tap hold and then swipe horizontally. Here is a video I found on this: https://www.youtube.com/watch?v=OvmkFfEbewg

In general, however, because of the swipe gesture that the slider requires, the control is not the most accessible (for example, to users with certain types of motor impairments). Therefore, using a stepper (plus minus buttons) as you suggested is a great workaround that is often more accessible.

**Comment by ismith78** [comment]
Hi Rachel,

I can confirm that the one finger vertical swipe up/down is consistent across iOS / Android / Kindle. However, this requires quite a bit of control, so I think the addition of the plus and minus buttons will help those who are not as dextrous.

Many thanks for your help, I think the issue can be marked as resolved, pending any documentation in the accessibility section of Xamarin Forms that might be needed related to the use of a slider.

**Comment by rachelkang** [comment]
sounds great! happy to have helped, and glad you determined a better solution :)
